en/docs/learn/password-recovery-via-user-preferred-notification-channel.md

```diff
@@ -0,0 +1,597 @@
+# Password Recovery Via User Preferred Notification Channel
+
+This section guides you through setting up password recovery for users to recover a lost or 
+forgotten password.
+
+!!! Note
+    This feature is only available via Account Recovery REST APis. Currently, WSO2 IS does not 
+    support this feature via the User Portal.
+
+**Password Recovery Flow**
+
+ - User provides a set of unique claims to identify the user account.
+ - Then the server will prompt available notification channels to the user to receive username 
+ recovery notification.
+ - User selects a notification channel.
+ - Sever send the recovery notification to the user via the preferred notification channel.
+ - If the preferred channel is,
+    - Email: Click on the reset link in the email and reset the password.
+    - SMS: Provide the received One-Time Password(OTP) and reset the password. 
```

Comment:
- The user provides...

- Next, the server prompts the user to select from the available notification channels for recieving the username recovery notification.

- The user selects a notification channel. 

- The server sends the recovery notification....

- If the preferred channel...